i18n/zh-CN/docusaurus-plugin-content-docs-community/current/index.md

```diff
@@ -25,10 +25,10 @@ Our formal announcement will be on our forums.
 
 This is the main method to communicate with Beyond Storage developers.
 
-We host our matrix server on `https://chat.aos.dev`.
+We host our matrix server on `https://chat.matrix.org`.
```

Comment:
Please don't update files under i18n.

Reply:
My fault, I just search it globally and modify all of them.